### PR TITLE
Prepare for publishing to NPM

### PR DESCRIPTION
**Pull request by NoahAndrews**

### CODE REVIEW

**Comment by sumigrev** [comment]
Can you push the changes to the `package-lock.json` file?

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This package enables Node.js applications to use a CAN bus over USB.
 
 ## Making a release
 
-1. Update `version` field in `package.json`
-2. Push to GitHub
-3. Run `npm run publish`
+1. Check out the `main` branch
+2. Update `version` field in `package.json`
```
Comment (sumigrev) [inline README.md:8]: I would add that they should also run `npm install`, so that the version numbers between `package.json` and `package-lock.json` match

Comment (NoahAndrews) [reply README.md:8]: Good call

**Review by sumigrev — APPROVED** [review]
Awesome!

**Pull request merged**